packages/preact-iso/router.js

```diff
@@ -19,27 +19,28 @@ const UPDATE = (state, url, push) => {
 };
 
 export const exec = (url, route, matches) => {
-	url = url.trim('/').split('/');
-	route = (route || '').trim('/').split('/');
+	url = url.replace(/(^\/+|\/+$)/g, '').split('/');
+	route = (route || '').replace(/(^\/+|\/+$)/g, '').split('/');
```

Comment:
Not sure if this is smaller:

```suggestion
	url = url.split('/').filter(Boolean);
	route = (route || '').split('/').filter(Boolean);
```